internal/test_helpers/scenarios/pass_base/app/main.py

```
@@ -0,0 +1,202 @@
+        except StopIteration:
+            # Returns "" if header_name key is not found
+            return ""
+
+
+class Response:
+    version = "HTTP/1.1"
+
+    def __init__(
+        self,
+        status: HTTPStatus = HTTPStatus.OK,
+        headers: dict[str, str] = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)